### ISSUE TITLE
Is there a reason to add a blank line after a given?

### ISSUE THREAD

**Issue by murdos** [mapado/prettier-plugin-gherkin#15]
I was going to apply this plugin to my projects (thanks BTW!), until I noticed that blank lines were added systematically after given phrases, which lead to a strange formatting: why make a separate block for given, but not separate when and then blocks?

### Before
```feature
  Scenario: Should not add beer as user
    Given I am logged in as "user"
    When I add beer to catalog
      | Name       | Cloak of feathers |
      | Unit price | 8.53              |
    Then I should be forbidden
```

### After
```feature
  Scenario: Should not add beer as user
    Given I am logged in as "user"

    When I add beer to catalog
      | Name       | Cloak of feathers |
      | Unit price | 8.53              |
    Then I should be forbidden
```

**Comment by jdeniau** [comment]
Hi,

Yes I did that on purpose in order to separate the "fixtures" bloc and the real test bloc.

In your case you do have only one given, but you can have something like that :

```gherkin
Given xxx
And yyy
And zzz

When I do aaa
Then bbb
And ccc
```

I think it does improve the readability of the test file.

**Comment by murdos** [comment]
Do you think that it could it could be turned off with a configuration parameter?
I understand your POV, but this is still quite unusual and controversial :)

**Comment by jdeniau** [comment]
It might be yes. I will think about it but it may also become the default value :

It you have more than one line break, it will remove extra line break to be only one. 
If you do not have line break, then the plug-in will keep "as-it" unless you configure the option `lineBreakAfterContext` to `"force"` or something like that. In this case, it will force a line break (as it does now).

**Comment by jdeniau** [comment]
@murdos this has been released as the default value in version 2.2.1 (2.2.0 + two small fixes when I tested on our stack)

See https://github.com/mapado/prettier-plugin-gherkin/blob/main/CHANGELOG.md#221---2023-12-28

**Comment by murdos** [comment]
@jdeniau: thanks! :heart:
